integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/BigQueryNodeVisitor.java

```diff
@@ -22,12 +24,14 @@
  * bigquery://&lt;projectId&gt;.&lt;.datasetId&gt;.&lt;tableName&gt;</code> . The namespace for
  * bigquery tables is always <code>bigquery</code> and the name is the FQN.
  */
-public class BigQueryNodeVisitor extends QueryPlanVisitor<LogicalPlan, OpenLineage.Dataset> {
```

Comment:
_Minor_: You may want to update the class javadocs as the class definition has change? That is, `D` can either be an input or output dataset.